Enqueue unlimited

tarakhanal opened the pull request:
Added the auto option for Max Item Count port and updated the documentation on vuo.list.enqueue.md accordingly.

tarakhanal committed:
Add auto option to denote unlimited for maxItemCount variable

tarakhanal committed:
Update documentation for auto option on Enqueue node

jlstrecker commented:
Here's the patch I mentioned that makes "auto":infinity work for VuoInteger:
[auto-infinity.txt](https://github.com/vuo/vuo/files/5928353/auto-infinity.txt)

tarakhanal committed:
Allow 'auto' values of infinity and -infinity in port metadata

In port metadata, allow ‘auto’ values of infinity and -infinity for VuoInteger ports (similar to what is already allowed for VuoReal ports).

jlstrecker commented:
3 more patches:

[enqueue-unlimited-description.txt](https://github.com/vuo/vuo/files/5939928/enqueue-unlimited-description.txt) — Update and clarify node documentation for unlimited queues.

[enqueue-unlimited-version.txt](https://github.com/vuo/vuo/files/5939938/enqueue-unlimited-version.txt) — Increment version number.

[enqueue-unlimited-test.txt](https://github.com/vuo/vuo/files/5939945/enqueue-unlimited-test.txt) — Add a test of the Enqueue node.

tarakhanal committed:
Update and clarify node documentation for unlimited queues

tarakhanal committed:
Increment version number

tarakhanal committed:
Add a test of the Enqueue node

smokris commented:
Thanks, @tarakhanal and @jlstrecker!  I've merged this.
